gatsby/src/docs/workflow/alerts-notifications/alerts.mdx

```diff
@@ -36,7 +36,7 @@ As mentioned [above](#for-errors), a metric is the value of an aggregate functio
 
 A metric alert has, at most, two triggers. The first is a critical trigger, which is required. The second is a warning trigger, which is optional. Triggers are independent of one another; however, the warning (optional) must be reached before the critical (required) trigger.
 
-Sentry evaluates triggers approximately every minute from the highest severity to lowest. Sentry creates an alert with the severity of the matched trigger (warning or critical). If an alert is already active, its status is updated. Admins can resolve alerts manually or automatically by setting the resolution threshold.
+Sentry evaluates triggers approximately every minute from the highest severity to lowest. Sentry creates an alert with the severity of the matched trigger (warning or critical). If an alert is already active, its status is updated. If no resolution threshold is specified and the alert is neither a warning nor critical, the alert will automatically resolve. You can also resolve alerts manually.
```

Comment:
Not sure I understand "and the alert is neither a warning nor critical". Suggestion: "if no resolution threshold is specified, the alert will automatically resolve when it's not violating the critical or warning conditions any more."